dMasterServer/MasterServer.cpp

```diff
@@ -154,6 +154,15 @@ int main(int argc, char** argv) {
 	const bool cdServerExists = std::filesystem::exists(BinaryPathFinder::GetBinaryDir() / "resServer" / "CDServer.sqlite");
 	const bool oldCDServerExists = std::filesystem::exists(Game::assetManager->GetResPath() / "CDServer.sqlite");
 	const bool fdbExists = std::filesystem::exists(Game::assetManager->GetResPath() / "cdclient.fdb");
+	const bool resServerPathExists = std::filesystem::is_directory(BinaryPathFinder::GetBinaryDir() / "resServer");
+
+	if (!resServerPathExists) {
+		LOG("%s does not exist, creating it.", (BinaryPathFinder::GetBinaryDir() / "resServer").c_str());
```

Comment:
Lets do this concatenation of binary dir and resServer once
also .c_str on a std::filesystem::path returns a wstring on windows, so this logger call should be .string().c_str()

Reply:
was following the convention below with the .c_str()

Reply:
yes and that is incorrect and should be changed as well.  Unix uses std::string for its file paths and windows uses std::wstring for its